### PR TITLE
Milestone 3: Add Footer

### PR DESCRIPTION
**Pull request by developerwaleed**
- Footer Added.
- Styling Done.
- Added Form.
- All the fonts, colors and design is from "FIGMA TEMPLATE 3".

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -346,6 +345,47 @@ <h2>Skills</h2>
...
+  
+  
+  
+    <footer>
+        <div>
+            <p id="footer-text">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+        </div>
+        <form action="./index.html">
```
Comment (danmainah) [inline index.html]: - [ ] Kindly make sure your form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html)

File: index.html [hunk 2]
```diff
@@ -346,6 +345,47 @@ <h2>Skills</h2>
...
+            <label>
+                <input type="text" name="user" placeholder="Full name">
+            </label>
+            <label>
+                <input type="email" name="email" placeholder="Email Address">
+            </label>
+            <label>
+                <textarea>Email text here</textarea>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly make sure has the `maxlength="500"` attribute as specified in the requirements also correct the message displayed in the text area, it is not emailed text here, but message text here.

File: index.html [hunk 3]
```diff
@@ -346,6 +345,47 @@ <h2>Skills</h2>
...
+  
+    <footer>
+        <div>
+            <p id="footer-text">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+        </div>
+        <form action="./index.html">
+            <label>
+                <input type="text" name="user" placeholder="Full name">
```
Comment (danmainah) [inline index.html]: - [ ] Kindly make sure the name input has `maxlength="30"` attribute as specified in the design.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @caasperr and @Sahar-SE , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] Kindly make sure to add image into your text area to match with the design given.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


- Kindly ensure that as  partners you always have balanced commits in pair programming project 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**